test/js/pageobjects/DocumentUpload.js

```diff
@@ -9,7 +20,91 @@ class DocumentUpload extends Base{
       return input
     })()}
 
-    getUploadButton() { return this.$('.onfido-sdk-ui-Uploader-button')}
+    upload(filename) {
+      const input = this.$('.onfido-sdk-ui-CustomFileInput-input')
+      const pathToTestFiles = '../../features/helpers/resources/'
+      const sendKeysToElement = input.sendKeys(path.join(__dirname, pathToTestFiles + filename))
+      return sendKeysToElement
+    }
+
+    /* eslint-disable no-undef */
+    copy(lang) { return locale(lang) }
+
+    verifyCrossDeviceUIElements() {
+      const documentUploadScreenCrossDeviceStrings = copy(lang).cross_device.switch_device
```

Comment:
this type of abstraction is what would make sense to have in the method `copy`. You should return only copy relevant to this page in that method